ng2-components/ng2-activiti-processlist/src/services/activiti-process.service.ts

```diff
@@ -72,13 +72,17 @@ export class ActivitiProcessService {
 
     /**
      * Retrieve the process filter by id
-     * @param processId - string - The id of the filter
+     * @param filterId - number - The id of the filter
+     * @param appId - string - optional - The id of app
      * @returns {Observable<FilterProcessRepresentationModel>}
      */
-    getProcessFilterById(processId: string, appId?: string): Observable<FilterProcessRepresentationModel> {
-        return Observable.fromPromise(this.callApiGetUserProcessInstanceFilters(appId))
+    getProcessFilterById(filterId: number, appId?: string): Observable<FilterProcessRepresentationModel> {
+        let filterObj = {
+            'appId': appId
+        };
+        return Observable.fromPromise(this.callApiGetUserProcessInstanceFilters(filterObj))
```

Comment:
Please, can you use this method 
callApiProcessFilters 

Can you also remove this method callApiGetUserProcessInstanceFilters we don't need it